### PR TITLE
issue template for model compression/speedup

### PR DESCRIPTION
**Pull request by Lijiaoa**
### Description ###
related issue #5156

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/model-compression-bug-report.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+**Describe the bug**:
+
+
+
+**Environment**:
+- NNI version:
+- Training service (local|remote|pai|aml|etc):
+- Python version:
+- PyTorch version:
+- Cuda version:
```
Comment (Louis-J) [inline .github/ISSUE_TEMPLATE/model-compression-bug-report.md:17]: use`cpu or cuda version`?

File: .github/ISSUE_TEMPLATE/enhancement-compression.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+---
```
Comment (Louis-J) [inline .github/ISSUE_TEMPLATE/enhancement-compression.md:1]: suggestion: use consistent filename style, such as 'model-compression-bug-report' and 'model-compression-enhancement', or 'bug-report-model-compression' and 'enhancement-model-compression'

Comment (Lijiaoa) [reply .github/ISSUE_TEMPLATE/enhancement-compression.md:1]: ok!

File: .github/ISSUE_TEMPLATE/model-compression-bug-report.md (hunk 1 above)
Comment (Lijiaoa) [reply .github/ISSUE_TEMPLATE/model-compression-bug-report.md:17]: So need to add a new line `- Cup version:` or change `- Cuda version` to `- Cpu or cuda version`

Comment (Louis-J) [reply .github/ISSUE_TEMPLATE/model-compression-bug-report.md:17]: i think it's better to change it to ` Cpu or cuda version`

**Review by J-shang — APPROVED** [review]

**Review by Louis-J — APPROVED** [review]

**Pull request merged**